spec/std/file_spec.cr

```diff
@@ -279,6 +279,7 @@ describe "File" do
     File.dirname("/Users/foo/bar.cr").should eq("/Users/foo")
     File.dirname("foo").should eq(".")
     File.dirname("").should eq(".")
+    File.dirname("/τελεία/τελεία").should eq("/τελεία")
```

Comment:
IMO it would be better to have a different path parts, in order to make sure that a proper piece is returned it - as now it could any of those.

Reply:
Good idea. Thanks!